Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -1175,6 +1175,25 @@ class TabsSetting: Setting {
     }
 }
 
+class NotificationsSetting: Setting {
+    override var accessoryView: UIImageView? { return SettingDisclosureUtility.buildDisclosureIndicator(theme: theme) }
+
+    override var accessibilityIdentifier: String? { return "NotificationsSetting" }
```

Comment:
we have an `AccessibilityIdentifiers` struct to move this kind of stuff into so we can keep our code easily writeable. :)